diff --git sesman/sesman.ini sesman/sesman.ini
index 3d09007..0b42f0d 100644
--- sesman/sesman.ini
+++ sesman/sesman.ini
@@ -81,15 +81,13 @@ param6=-dpi
 param7=96
 
 [Xorg]
-param0=Xorg
+param0=Xorg.xrdp
 param1=-config
 param2=xrdp/xorg.conf
-param3=-logfile
-param4=/dev/null
-param5=-noreset
-param6=-ac
-param7=-nolisten
-param8=tcp
+param3=-noreset
+param4=-ac
+param5=-nolisten
+param6=tcp
 
 [Chansrv]
 # drive redirection, defaults to xrdp_client if not set
