diff --git sesman/sesman.ini sesman/sesman.ini
index 140cc49..22dc4bf 100644
--- sesman/sesman.ini
+++ sesman/sesman.ini
@@ -78,14 +78,14 @@ param=-dpi
 param=96

 [Xorg]
-param=Xorg
+param=Xorg.xrdp
 param=-config
 param=xrdp/xorg.conf
 param=-noreset
 param=-nolisten
 param=tcp
-param=-logfile
-param=/dev/null
+#param=-logfile
+#param=/dev/null

 [Chansrv]
 ; drive redirection, defaults to xrdp_client if not set

