diff --git sesman/sesman.ini sesman/sesman.ini
index 5e70225..79bfde9 100644
--- sesman/sesman.ini
+++ sesman/sesman.ini
@@ -81,15 +81,15 @@ param=-dpi
 param=96

 [Xorg]
-param=Xorg
+param=Xorg.xrdp
 param=-config
 param=xrdp/xorg.conf
 param=-noreset
 param=-ac
 param=-nolisten
 param=tcp
-param=-logfile
-param=/dev/null
+#param=-logfile
+#param=/dev/null

 [Chansrv]
 # drive redirection, defaults to xrdp_client if not set
